framework/test/unit/modules/interoperability/internal_method.spec.ts

```diff
@@ -1288,8 +1479,45 @@ describe('Base interoperability internal method', () => {
 				expect.toBeObject() as Certificate,
 			);
 		});
+
+		it('should resolve when validatorsHash in certificate and state store are equal', async () => {
+			const certificate: Certificate = {
+				...defaultCertificate,
+				timestamp: 1000,
+			};
+
+			await interopMod.stores.get(ChainAccountStore).set(methodContext, txParams.sendingChainID, {
+				lastCertificate: {
+					height: 100,
+					timestamp: 10,
+					stateRoot: cryptoUtils.getRandomBytes(HASH_LENGTH),
+					validatorsHash: defaultCertificate.validatorsHash,
+				},
+				name: 'rand',
+				status: 0,
+			});
+
+			const encodedCertificate = codec.encode(certificateSchema, certificate);
+			await expect(
+				mainchainInteroperabilityInternalMethod.verifyCertificate(
+					methodContext,
+					{
+						...txParams,
+						certificate: encodedCertificate,
+					},
+					1001,
+				),
+			).resolves.toBeUndefined();
+			expect(validator.validate).toHaveBeenCalledWith(
+				certificateSchema,
+				expect.toBeObject() as Certificate,
+			);
+		});
+
+		it('should resolve when validatorsHash are NOT equal, but validators are updated', async () => {});
 	});
 
+	// TODO: test where the validator list in the validators store is NOT sorted
```

Comment:
this is DONE

Reply:
Edited [here](https://github.com/LiskHQ/lisk-sdk/blob/4cbdadaf90ceccfe5fdf242f688ee3d30da90c4d/framework/test/unit/modules/interoperability/internal_method.spec.ts#L1531)